NEWS.md

```diff
@@ -1,5 +1,271 @@
+  current API.
+
+  [Unicode 16.0]: https://www.unicode.org/versions/Unicode16.0.0/
+
+### New
+
+- Implemented the `GroupLatch` action, usually activated with the keysym
+  `ISO_Group_Latch`.
+  ([#455](https://github.com/xkbcommon/libxkbcommon/issues/455))
+
+- Symbols: Added support for *multiple actions per levels:*
+  - When no action is specified, `interpret` statements are used to find an
+    action corresponding for *each* keysym, as expected.
+  - When both keysyms and actions are specified, they may have a different count
+    for each level.
+  - For now, at most one action of each following categories is allowed per level:
+    - modifier actions: `SetMods`, `LatchMods`, `LockMods`;
+    - group actions: `SetGroup`, `LatchGroup`, `LockGroup`.
+
+    Some examples:
+    - `SetMods` + `SetGroup`: ok
+    - `SetMods` + `SetMods`: error
+    - `SetMods` + `LockMods`: error
+    - `SetMods` + `LockGroup`: ok
+
+  ([#486](https://github.com/xkbcommon/libxkbcommon/issues/486))
+
+- Updated keysyms using latest [xorgproto]
+  (commit: `d7ea44d5f04cc476dee83ef439a847172f7a6bd1`):
+
+    Additions:
+
+    - `XKB_KEY_XF86RefreshRateToggle`
+    - `XKB_KEY_XF86Accessibility`
+    - `XKB_KEY_XF86DoNotDisturb`
+
+    Relevant upstream merge request: [xorgproto-91].
+
+  [xorgproto-91]: https://gitlab.freedesktop.org/xorg/proto/xorgproto/-/merge_requests/91
+
+- Added deprecated keysym warnings:
+  - *name* deprecation (typo, historical alias), reporting the reference name;
+  - *keysym* deprecation (ambiguous meaning, all names deprecated).
+
+  These warnings are activated by setting the log verbosity to at least 2.
+
+  It is advised to fix these warnings, as the deprecated items may be removed in
+  a future release.
+
+- `xkbcommon-names.h`: Added the following modifiers names definitions:
+  - `XKB_MOD_NAME_MOD1`
+  - `XKB_MOD_NAME_MOD2`
+  - `XKB_MOD_NAME_MOD3`
+  - `XKB_MOD_NAME_MOD4`
+  - `XKB_MOD_NAME_MOD5`
+  - `XKB_VMOD_NAME_ALT`
+  - `XKB_VMOD_NAME_META`
+  - `XKB_VMOD_NAME_NUM`
+  - `XKB_VMOD_NAME_SUPER`
+  - `XKB_VMOD_NAME_HYPER`
+  - `XKB_VMOD_NAME_LEVEL3`
+  - `XKB_VMOD_NAME_LEVEL5`
+  - `XKB_VMOD_NAME_SCROLL`
+
+- `xkbcommon-names.h`: Added `XKB_LED_NAME_COMPOSE` and `XKB_LED_NAME_KANA`
+  definitions to cover all LEDs defined in
+  [USB HID](https://usb.org/sites/default/files/hid1_11.pdf).
+
+  Contributed by Martin Rys
+
+- Rules: Use XKB paths to resolve relative paths in include statements.
+  ([#501](https://github.com/xkbcommon/libxkbcommon/issues/501))
+
+- Rules: Added support for special layouts indexes:
+  - *single*: matches a single layout; `layout[single]` is the same as without
+    explicit index: `layout`.
+  - *first*: matches the first layout/variant, no matter how many layouts are in
+    the RMLVO configuration. Acts as both `layout` and `layout[1]`.
+  - *later*: matches all but the first layout. This is an index range. Acts as
+    `layout[2]` .. `layout[4]`.
```

Comment:
Maybe not mention the `4` since we may increase it in the future?

Reply:
Good point, I changed to `MAX_LAYOUT` and indicate it is currently 4.